web/server/codechecker_server/api/report_server.py

```diff
@@ -149,6 +149,29 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def parse_codechecker_review_comment(source_file_name,
+                                     report_line,
+                                     checker_name):
+    """Parse the CodeChecker review comments from a source file at a given
+    position.  Returns an empty list if there are no comments.
+    """
+    src_comment_data = []
+    with open(source_file_name,
+              encoding='utf-8',
+              errors='ignore') as sf:
+        if contains_codechecker_comment(sf):
+            sc_handler = SourceCodeCommentHandler()
+            try:
+                src_comment_data = sc_handler.filter_source_line_comments(
+                    sf,
+                    report_line,
+                    checker_name)
+            except SpellException as ex:
+                LOG.warning(f"File {source_file_name} contains {ex}")
+                return []
```

Comment:
Do we need this return statement here? In case of exception we will return with an empty array at the end of this function: `return src_comment_data`